modules/cockroach/src/main/java/org/testcontainers/containers/CockroachContainer.java

```diff
@@ -0,0 +1,86 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.HttpWaitStrategy;
+
+import java.time.Duration;
+
+public class CockroachContainer<SELF extends CockroachContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
+    public static final String NAME = "cockroach";
+    public static final String IMAGE = "cockroachdb/cockroach";
+    public static final int REST_API_PORT = 8080;
+    public static final int DB_PORT = 26257;
+
+    private String databaseName = "test";
+    private String username = "root";
+    private String password = "";
+
+    public CockroachContainer() {
+        this(IMAGE + ":v1.1.2");
```

Comment:
Since the IMAGE is already a constant, we can put the default TAG in a constant as well :slightly_smiling_face: